src/indicators/flooding/glofas/utils.py

```diff
@@ -184,7 +184,7 @@ def get_groups_above_threshold(observations, threshold, min_duration=1):
     groups = np.where(np.diff(observations > threshold, prepend=False, append=False))[
```

Comment:
Only if not too much effort, but maybe change the function name `get_groups_above_threshold` to `get_groups_ge_threshold` ?